DEVELOPING.md

```diff
@@ -0,0 +1,129 @@
+
+To change which tests run and which are ignored, set the `liquibase.sdk.testSystem.test` attribute to be a comma-separated list. This can be most easily set in your liquibase.sdk.local.yml file (see below).
+
+The `liquibase.sdk.testSystem.test` setting can also specify specific settings for the test systems you wish to test against. 
+For example, if you have `liquibase.sdk.testSystem.test=h2?version=1.4.200,mysql`, when your tests call `getTestSystem("h2")` it will get an h2 version 1.4.200 system. You can also specify profiles (see below) as well, like `liquibase.sdk.testSystem.test=mssql:case-sensitive,mysql`    
+
+#### Starting test systems independently from tests
+
+Often times you need to have a database you can manually run liquibase against outside of your automated tests. 
+The `liquibase-extension-testing.jar` and corresponding `liquibase-extension-testing-deps.jar` files add a `liquibase sdk system` command group to the Liquibase CLI which allow you to start and stop the same containers used by your test.
+By adding those jars to your LIQUIBASE_HOME/lib directory, you can run `liquibase sdk system up --name mysql` to bring up a mysql system, where the `name` argument is the name of the test system you want to start.
+
+To shut down the instances started this way, run `liquibase sdk system down --name mysql`
+
+#### Local configuration
+
+The liquibase.sdk.yaml file contains the default configuration, but there are times you may want to change the default behavior. 
+By creating a `liquibase.sdk.local.yml` file in the [same directory](liquibase-extension-testing/src/main/resources) as the liquibase.sdk.yaml file, you can add just the settings you want to override.
+NOTE: gitignore is set to not allow the file to be committed because it's designed to be specific to you.
+
+Below there is an example of a default configuration which is going to be used across the different profiles, unless any of them overrides any given property:
+```yaml
+liquibase:
+  sdk:
+    testSystem:
+      default:
+          username: lbuser
+          password: LiquibasePass1
+          catalog: lbcat
+          altCatalog: lbcat2
+          altSchema: lbschem2
+          altTablespace: liquibase2
+          keepRunning: true         
+      test: mysql,h2,mssql     
+      acceptLicenses: mssql
+```
+will run tests against mysql, h2, and mssql. It also specifies that you accept the EULA license the mssql docker container requires you to accept.
+
+##### Test System Profiles
+
+In both the standard liquibase.sdk.yaml and liquibase.sdk.local.yaml files, "profiles" can be defined which override specific settings from the default configuration.
+For example, the h2 configuration defines a "1.x" and a "2.x" profile:
+
+```yaml
+  h2:
+    url: jdbc:h2:mem:${catalog};DB_CLOSE_DELAY=-1
+    profiles:
+      "1.x":
+        version: 1.4.200
+      "2.x":
+        version: 2.0.206
+```
+which makes it easier for users who just care about the top-level version. 
```

Comment:
Edit at line 128:

"The advantage of profiles is it allows users to specify the exact version of h2 relevant for their testing."

Reply:
Done.